content/topics/quality-education/Stories/Anushka-Vuppala-story.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Education in Bolivia
+author: Anushka Vuppala
+date: 2023-04-29
+data: "https://raw.githubusercontent.com/DSSD-Madison/Nagoya/main/data/GeoDS4Bolivia.geojson"
+categories: ['Education', 'LiteracyRate', 'Teachers']
+---
+
+## **Variables Used :**
+| Variable | Description |
+|----------|-------------|
+| sdg4_4_phe | Population with higher education (>= 19 years), 2012 (%) |
+| sdg4_6_lr | Literacy rate for (>= 15 years), 2012 (%) |
+| sdg4_c_qti | Qualified teachers at the initial level, 2016 (%) |
+| sdg4_c_qts | Qualified teachers at the secondary level, 2016 |
+| sdg9_5_eutf | Educational units with technological floors, 2016 (%) |
+| sdg9_5_cd | Kuaa computers delivered, 2016 (per 100 school-age population, 6-19 years) |
+| dep | Bolivia States |
+
+
+Bolivia, a high-altitude country located in the Andes Mountains, offers visitors a unique blend of indigenous cultures, colonial architecture, and breathtaking natural beauty. However, despite its many strengths, Bolivia faces significant challenges in its education system. In the story that follows, we will take a journey through the education system, exploring different aspects that affect the students and teachers. Our journey begins in 2012, where we delve into the literacy rate of different states of Bolivia.
+
+{{< image class="medium" src="/topics/quality-education/Stories/images/IMG1.png" alt="Literacy Rate with Population with higher education" >}}
+
+Despite having a high literacy rate, only a small percentage of the population had a higher level of education, which could be attributed to poverty and a lack of access to education. This would indicate that the population knows how to read and write but very few people in the population attended college or university. The main reason behind this could be poverty. Many children have to go to work and support their impoverished family rather than go to school. <em>(Top 4 Reasons Education in Bolivia Lags, 2015) </em>
```

Comment:
For safety, raw HTML is not rendered in the Markdown documents. The <em> tag instead use the Markdown *italics* (`*italics*`)

If there's something that isn't supported by Markdown/existing shortcodes, let the team know and we can works towards implementing more!